config/schedule.rb

```diff
@@ -6,10 +6,14 @@
   runner 'scripts/mail_uncaptured_donations.rb'
 end
 
-every 6.hours do
-  runner 'scripts/recalc_abilities.rb'
-end
-
 every 1.day, at: '02:10' do
   runner 'scripts/prune_email_logs.rb'
 end
+
+every 1.day, at: '02:15' do
```

Comment:
I assume this script runs as System?  (Asking for attribution purposes on the delete.)

Reply:
It would if I'd remembered to do that...